redbot/cogs/mod/kickban.py

```diff
@@ -210,15 +210,19 @@ class KickBanMixin(MixinMeta):
         self,
         ctx: commands.Context,
         user: discord.Member,
-        days: Optional[int] = 0,
+        days: Optional[int] = None,
         *,
         reason: str = None,
     ):
         """Ban a user from this server and optionally delete days of messages.
 
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
-
+        guild = ctx.guild
+        if days == None:
```

Comment:
You should use `if days is None` instead, because there's only one `None` object.

Reply:
Done.